docs/docs/plugins.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Juno Plugins
```

Comment:
do you think it's worth to mention separate repo with plugin example ?

Reply:
Currently it's private, so I just opted for placing the plugin example in the readme. Happy either way though